packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -239,7 +247,10 @@ export default async (cmd: Command) => {
   }
 
   const answers: Answers = await inquirer.prompt(questions);
-  const pluginId = cmd.backend ? `${answers.id}-backend` : answers.id;
+  const pluginId =
+    cmd.backend && !answers.id.match(/-backend$/)
```

Comment:
Could use `.endsWith()` here to avoid regex when possible :grin:

Reply:
Good point!